src/axom/mint/mesh/RectilinearMesh.hpp

```diff
@@ -7,8 +7,9 @@
 #define MINT_RECTILINEARMESH_HPP_
 
 #include "axom/config.hpp"
-#include "axom/mint/mesh/StructuredMesh.hpp"  // base class
-#include "axom/mint/config.hpp"               // for compile-time definitions
+#include "axom/mint/mesh/StructuredMesh.hpp"
+#include "axom/mint/config.hpp"
+#include "axom/export/mint.h"
```

Comment:
@white238 -- here's an example of including the exported header.

Here's what's generated for export/mint.h` on toss3
```

#ifndef AXOM_MINT_EXPORT_H
#define AXOM_MINT_EXPORT_H

#ifdef AXOM_MINT_STATIC_DEFINE
#  define AXOM_MINT_EXPORT
#  define AXOM_MINT_NO_EXPORT
#else
#  ifndef AXOM_MINT_EXPORT
#    ifdef mint_EXPORTS
        /* We are building this library */
#      define AXOM_MINT_EXPORT 
#    else
        /* We are using this library */
#      define AXOM_MINT_EXPORT 
#    endif
#  endif

#  ifndef AXOM_MINT_NO_EXPORT
#    define AXOM_MINT_NO_EXPORT 
#  endif
#endif

#ifndef AXOM_MINT_DEPRECATED
#  define AXOM_MINT_DEPRECATED __attribute__ ((__deprecated__))
#endif

#ifndef AXOM_MINT_DEPRECATED_EXPORT
#  define AXOM_MINT_DEPRECATED_EXPORT AXOM_MINT_EXPORT AXOM_MINT_DEPRECATED
#endif

#ifndef AXOM_MINT_DEPRECATED_NO_EXPORT
#  define AXOM_MINT_DEPRECATED_NO_EXPORT AXOM_MINT_NO_EXPORT AXOM_MINT_DEPRECATED
#endif

#if 0 /* DEFINE_NO_DEPRECATED */
#  ifndef AXOM_MINT_NO_DEPRECATED
#    define AXOM_MINT_NO_DEPRECATED
#  endif
#endif

#endif /* AXOM_MINT_EXPORT_H */
```